CHANGELOG.md

```diff
@@ -1,4 +1,29 @@
-## 0.5.1 (Unreleased)
+## 1.0.0 (Unreleased)
+FEATURES:
+
+* The resource schema is modified to have more attributes for the vra7_deployment resource
+* Feature request: Day 2: Change the number of VMs created by the vRA blueprint ([#47](https://github.com/terraform-providers/terraform-provider-vra7/issues/47))
+* Does 'import' work? ([#29](https://github.com/terraform-providers/terraform-provider-vra7/issues/29))
```

Comment:
Might want to comment on the base feature for these two: Implement "terraform import" capability